### DISCUSSION TITLE
Assistants not working in v0.7.6-rc1

### DISCUSSION THREAD

**Discussion by ByungsunAhn** [danny-avila/LibreChat#4908] [Troubleshooting] [answered]
When I update from HuggingFace via Factory Rebuild, the Assistant API connection seems to have been lost.
In the first sentence of the response, it says
"I'm unable to access the contents of the uploaded files at the moment."
It seems to be a problem with the actual connection to the API, as only the guide to the instruction seems to be passed through.

**Comment by danny-avila** [comment, marked as answer]
It should be fixed here: https://github.com/danny-avila/LibreChat/pull/4912

Also I recommend using Agents going forward instead: https://www.librechat.ai/docs/features/agents
